tests/migrations/test_commands.py

```diff
@@ -526,6 +526,7 @@ def test_sqlmigrate_forwards(self):
         sqlmigrate outputs forward looking SQL.
         """
         out = io.StringIO()
+        connection.can_rollback_ddl = True
```

Comment:
This should not be added; instead the assertions below should be adjusted to skip `connection.ops.start_transaction_sql().lower()` lookup `if not connection.features.can_rollback_ddl`.

Reply:
I thought of doing that, but didn't really knew if that was a right thing to do